vcluster/integrations/custom-plugins.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Custom Plugins
+sidebar_label: Custom Plugins
+sidebar_class_name: host-nodes
+sidebar_position: 4
+---
+
```

Comment:
Maybe adding an overview section explaining what custom plugins are and why users would create them would help readers understand the purpose before diving into SDK documentation